litert/vendors/intel_openvino/compiler/graph_iterator.h

```diff
@@ -87,7 +87,10 @@ class GraphIteratorDelegate
   /// If there is no query for specific sub-graph iterator shouldn't be created
   /// idx should be in range 0..get_subgraph_size()-1
   std::shared_ptr<ov::frontend::tensorflow_lite::GraphIterator> get_subgraph(
-      size_t idx) const override {};
+      size_t idx) const override {
+    LITERT_LOG(LITERT_ERROR, "get_subgraph not implemented");
```

Comment:
Looks like a style formatting issue here

Reply:
It has already used `clang-format  -style=google`.